tests/integration/callbacks/test_arbitrary_callbacks.py

```diff
@@ -165,3 +165,26 @@ def on_click(clicked):
         ".dash-fe-error__title",
         "Callback error with no output from input start.n_clicks",
     )
+
+
+def test_arb006_multi_set_props(dash_duo):
+    app = Dash()
+
+    app.layout = [
+        html.Button("start", id="start"),
+        html.Div("initial", id="output"),
+    ]
+
+    @app.callback(
+        Input("start", "n_clicks"),
+    )
+    def on_click(_):
+        set_props("output", {"children": "changed"})
+        set_props("output", {"style": {"background": "rgb(255,0,0)"}})
```

Comment:
this example doesn't overwrite an existing property, so if `{**updates, **props}` is actually a bug, this test won't trigger it.